protocol/scripts/genesis/prod_pregenesis.sh

```diff
@@ -0,0 +1,173 @@
+#!/bin/bash
+set -eo pipefail
+
+# This script creates the pregenesis file for a production network.
+#
+# The script must be run from the root of the `v4-chain` repo.
+#
+# example usage:
+# $ ./scripts/genesis/prod_pregenesis.sh ./build/dydxprotocold
+
+# Check for missing required arguments
+if [ -z "$1" ]; then
+  echo "Error: Missing required argument DYDX_BINARY."
+  echo "Usage: $0 <DYDX_BINARY> [-s|--SEED_FAUCET_USDC]"
+  exit 1
+fi
+
+# Capture the required argument
+DYDX_BINARY="$1"
+
+TMP_CHAIN_DIR="/tmp/prod-chain"
+TMP_EXCHANGE_CONFIG_JSON_DIR="/tmp/prod-exchange_config"
+FIFTEEN_ZEROS="000000000000000"
```

Comment:
Originally added `FIFTEEN_ZEROS` because I wanted to add a some value for `100$FIFTEEN_ZEROS` which turned out wasn't necessary. IMO it's probably cleaner to just define one constant of `$EIGHTEEN_ZEROS`